addons/markers/functions/fnc_onButtonClickConfirm.sqf

```diff
@@ -17,12 +17,27 @@
 params ["_buttonOk"];
 
 private _display = ctrlParent _buttonOk;
-private _description = _display displayctrl IDC_INSERT_MARKER;
+private _description = _display displayCtrl IDC_INSERT_MARKER;
 private _aceTimestamp = _display displayCtrl IDC_ACE_INSERT_MARKER_TIMESTAMP;
 
 // handle timestamp
 if (cbChecked _aceTimestamp && {ACE_player call FUNC(canTimestamp)}) then {
-    private _time = daytime;
+    private _time = switch (GVAR(timestampTimezone)) do {
+        case 1: {
+            systemTime params ["", "", "", "_hour", "_min", "_sec"];
+            _hour + _min/60 + _sec/3600
+        };
+        case 2: {
+            systemTimeUTC params ["", "", "", "_hour", "_min", "_sec"];
+            _hour = _hour + round (GVAR(timestampUTCOffset));
+            _min = _min + GVAR(timestampUTCMinutesOffset);
```

Comment:
`GVAR(timestampUTCMinutesOffset)` should take the sign of `GVAR(timestampUTCOffset)` into account.

E.g [UTC−09:30](https://en.wikipedia.org/wiki/UTC%E2%88%9209:30) needs to be set to `-10` and `30` for the settings right now, which I don't think is intuitive.

```suggestion
            _min = _min + (GVAR(timestampUTCMinutesOffset) * GVAR(timestampUTCOffset) / abs GVAR(timestampUTCOffset));
```

Reply:
Alternatively, you can add the negative offsets to the settings.

Reply:
Yeah thanks for pointing that out. It is definitly something I did not take in to account.

Reply:
`GVAR(timestampUTCOffset) / abs GVAR(timestampUTCOffset)` does not work with an offset of 0, becauese of devide by zero.